runtime/shared_common/CacheMap.cpp

```diff
@@ -2789,6 +2801,16 @@ SH_CacheMap::storeROMClassResource(J9VMThread* currentThread, const void* romAdd
 		result = resourceDescriptor->unWrap(resourceWrapper);
 	}
 
+	/* Update the min/max boundary with the stored metadata entry if allocated successfully */
+#if !defined(J9ZOS390) && !defined(AIXPPC)
+#if defined(LINUX)
+		if (J9_ARE_ALL_BITS_SET(*_runtimeFlags, J9SHR_RUNTIMEFLAG_ENABLE_PERSISTENT_CACHE))
+#endif
+		{
+			updateAccessedShrCacheMetadataBounds(currentThread, (uintptr_t *) result);
```

Comment:
This place and findCompiledMethod() call updateAccessedShrCacheMetadataBounds without checking that result is valid first.

Reply:
Already added checking code.